docs/tutorials/03_ground_state_solvers.ipynb

```diff
@@ -97,21 +113,21 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 10,
+   "execution_count": 5,
    "metadata": {},
    "outputs": [],
    "source": [
-    "from qiskit.aqua.algorithms import VQE\n",
+    "from qiskit.algorithms import VQE\n",
     "from qiskit.circuit.library import TwoLocal\n",
     "\n",
     "num_qubits = 4\n",
     "tl_circuit = TwoLocal(num_qubits, ['h', 'rx'], 'cz', \n",
     "                      entanglement='full', reps=3, parameter_prefix = 'y')\n",
     "\n",
-    "tl_circuit.draw(output = 'mpl')\n",
+    "tl_circuit.draw()\n",
```

Comment:
The mpl gives an arguably nicer output than the plain text draw. Either way I do not see any circuit in the output!

Reply:
fixed